src/main/java/au/org/emii/ncdfgenerator/NcdfDefinitionXMLParser.java

```diff
@@ -1,276 +1,374 @@
 
 package au.org.emii.ncdfgenerator;
```

Comment:
This file includes a lot of code.  Perhaps creating a parser sub package with separate files per class would make it easier to navigate through this.  There's a lot of files already at the default package level.